### PR TITLE
[Bulk] Validation-Skype-docs-Title-missing

### PR DESCRIPTION
**Pull request by Roudabeh1348**
This PR fixes 29 issues of missing titles in OfficeDev/Skype-docs repo.

### CODE REVIEW

File: Skype/UCWA/BatchingRequests.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-
+---
+title: batching requests for skype for business 2015
```
Comment (RedZephyr13) [inline Skype/UCWA/BatchingRequests.md:2]: Let's capitalize "batching."

File: Skype/UCWA/EventsInUCWA.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-
+---
+title: events in UCWA for skype for business 2015
```
Comment (RedZephyr13) [inline Skype/UCWA/EventsInUCWA.md:2]: Caps.

File: Skype/UCWA/EventsInUCWA.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+description: "ucwa 2.0 implements a server-to- client communication channel within HTTP with a long poll model called pending GET or P-GET."
```
Comment (RedZephyr13) [inline Skype/UCWA/EventsInUCWA.md:3]: More caps.

File: Skype/UCWA/MeTasks.md [hunk 4]
```diff
@@ -1,4 +1,7 @@
-
+---
+title: me tasks for skype for business 2015
+description: "Me tasks refers to topics showing code samples of tasks that are related to the **me** user, the currently logged-on user."
```
Comment (RedZephyr13) [inline Skype/UCWA/MeTasks.md:3]: I think we can leave out the aserisks around "me" here.

File: Skype/UCWA/changeNumber_ref.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+---
+title: ChangeNumber resource for Skype for Business 2015
+description: "changeNumber Changes or clears the number stored in the corresponding [phone](phone_ref.md) resource."
```
Comment (RedZephyr13) [inline Skype/UCWA/changeNumber_ref.md:3]: Capitalize plz.

File: Skype/UCWA/phoneDialInInformation_ref.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
+---
+title: PhoneDialInInformation for Skype for Business 2015
+description: "phoneDialInInformation represents phone access information for an onlineMeeting. Web links include rel and href."
```
Comment (RedZephyr13) [inline Skype/UCWA/phoneDialInInformation_ref.md:3]: Capitalize plz.

File: Skype/UCWA/sendProvisionalAnswer_ref.md [hunk 7]
```diff
@@ -1,3 +1,7 @@
+---
+title: SendProvisionalAnswer resource for Skype for Business 2015
+description: "sendProvisionalAnswer indicates a link in an audioVideoInvitation resource that lets the application send an answer for early media negotiation."
```
Comment (RedZephyr13) [inline Skype/UCWA/sendProvisionalAnswer_ref.md:3]: Capitalize plz.

File: Skype/UCWA/BatchingRequests.md (hunk 1 above)
Comment (Roudabeh1348) [reply Skype/UCWA/BatchingRequests.md:2]: fixed.

File: Skype/UCWA/DevelopingApplicationsWithUCWA.md [hunk 8]
```diff
@@ -1,4 +1,7 @@
-
+---
+title: developing applications with ucwa 2.0
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Skype/UCWA/DevelopingApplicationsWithUCWA.md:2]: done!

File: Skype/UCWA/DevelopingApplicationsWithUCWA.md [hunk 9]
```diff
@@ -1,4 +1,7 @@
...
+description: "how to develop applications for on-premises server and online server using the ucwa 2.0."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Skype/UCWA/DevelopingApplicationsWithUCWA.md:3]: fixed.

File: Skype/UCWA/EventsInUCWA.md (hunk 3 above)
Comment (Roudabeh1348) [reply Skype/UCWA/EventsInUCWA.md:3]: done!

File: Skype/UCWA/ImmediateForwardSettingsTarget_ref.md [hunk 10]
```diff
@@ -1,4 +1,8 @@
+---
+title: immediate forward settings target for skype for business 2015
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Skype/UCWA/ImmediateForwardSettingsTarget_ref.md:2]: fixed.

File: Skype/UCWA/MeTasks.md [hunk 11]
```diff
@@ -1,4 +1,7 @@
-
+---
+title: me tasks for skype for business 2015
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Skype/UCWA/MeTasks.md:2]: fixed.

File: Skype/UCWA/MeTasks.md (hunk 4 above)
Comment (Roudabeh1348) [reply Skype/UCWA/MeTasks.md:3]: removed.

File: Skype/UCWA/changeNumber_ref.md (hunk 5 above)
Comment (Roudabeh1348) [reply Skype/UCWA/changeNumber_ref.md:3]: done!

File: Skype/UCWA/conversationLogTranscripts_ref.md [hunk 12]
```diff
@@ -1,3 +1,7 @@
+---
+title: conversationLogTranscripts for Skype for Business 2015
+description: "conversationLogTranscripts refers to the archived messages from Exchange. Web links include rel and href."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Skype/UCWA/conversationLogTranscripts_ref.md:3]: fixed.

File: Skype/UCWA/conversationLogs_ref.md [hunk 13]
```diff
@@ -1,3 +1,7 @@
+---
+title: ConversationLogs for Skype for Business 2015
+description: "conversationLogs represents the user's past conversation logs which includes both peer-to-peers and conferences."
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Skype/UCWA/conversationLogs_ref.md:3]: done!

File: Skype/UCWA/phoneDialInInformation_ref.md (hunk 6 above)
Comment (Roudabeh1348) [reply Skype/UCWA/phoneDialInInformation_ref.md:3]: fixed.

File: Skype/UCWA/replayMessage_ref.md [hunk 14]
```diff
@@ -1,9 +1,12 @@
+---
+title: replayMessage reference for Skype for Business 2015
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply Skype/UCWA/replayMessage_ref.md:2]: Done!

File: Skype/UCWA/sendProvisionalAnswer_ref.md (hunk 7 above)
Comment (Roudabeh1348) [reply Skype/UCWA/sendProvisionalAnswer_ref.md:3]: fixed.

**Comment by Roudabeh1348** [comment]
@lindalu-MSFT The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. 
This will eliminate potential accessibility, security, and usability issues. 
This PR includes only build validation fixes and does not change other content. 

Please review within five business days and merge or comment in the PR with any changes you’d like to see. 
Thank you!

**Pull request merged**